view/frontend/web/js/instantsearch.js

```diff
@@ -1052,6 +1061,26 @@ define([
             };
         },
 
+        getRedirectWidget() {
+            const uiComponnent = this;
+            return {
+                render({ results }) {
+                    let content = '';
+                    if (results && results.renderingContent) {
+                        if (results.renderingContent.redirect) {
+                            if (!uiComponnent.hasInteracted) {
+                                window.location.href = results.renderingContent.redirect.url;
+                            }
+                            content += `<div class="instant-redirect">`;
+                            content += `<a href="${results.renderingContent.redirect.url}">${algoliaConfig.translations.redirectSearchPrompt} "${results.query}"</a>`;
```

Comment:
For an absolute URL, I would expect to have a `target="_blank"` attribute here but I'm not sure what is Algolia's stance regarding this aspect of the feature. (Could be added in the future config maybe ?)  Not blocking anyway.

Reply:
I wondered this as well. Right now the merch rules offer just a URL option. I think it's a good to provide a configurable option in the meantime for the target in the Magento side (dependent on the desired UX for the storefront - since you can redirect to both internal and external links).